exec/totemknet.c

```diff
@@ -63,6 +62,7 @@
 #include <qb/qbloop.h>
 #ifdef HAVE_LIBNOZZLE
 #include <libgen.h>
+#include <net/ethernet.h>
```

Comment:
OpenIndiana does not have net/ethernet.h, but we will get to it once libnozzle is working openindiana.

Reply:
Yup, indeed. I guess it still make sense to include file only when it is needed (= nozzle is compiled in) so that's the reason for simpler patch.